src/popcorn-list-movies/PopcornListMovies.js

```diff
@@ -66,6 +66,14 @@ export class PopcornListMovies extends LocalizeMixin(
     }
   }
 
+  isMovieImageValid(movie) {
+    if(movie.poster === "N/A") {

```

Comment:
I would modify this condition to include also scenarios in which movie.poster is not provided:

if (!movie.poster || movie.poster === "N/A")